.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+  - https://raw.githubusercontent.com/lessonly/rubocop-default-configuration/master/.rubocop.yml
```

Comment:
This is a centrally located shared rubocop config we can use in our distributed projects. Does this sound like a good idea? I know I'm going to need it on some of my other projects too 😆